prime-router/metadata/hl7_mapping/receivers/STLTs/CA/CA-receiver-transform.yml

```diff
@@ -90,11 +90,24 @@ elements:
 
   # SFT-1-2
   - name: software-vendor-name-type
-    condition: false
+    condition: true
```

Comment:
This approach makes me thing adding better support for `extend` is going to be needed sooner rather than later as I'm not sure this would work with a repeatable group/segment?

Reply:
Discussed on the call, this would not work for repeatable segments, but handling that is outside of scope of this work.